src/commands/registerCommands.ts

```diff
@@ -130,7 +130,8 @@ export function registerCommands(): void {
         node.setAllSubscriptionsLoaded();
         ext.azureTreeState.notifyChildrenChanged(node.id);
     });
-    registerCommand("azureResourceGroups.askAgentAboutActivityLog", askAgentAboutActivityLog);
+    registerCommandWithTreeNodeUnwrapping("azureResourceGroups.askAgentAboutActivityLog", async (context: IActionContext) => await askAgentAboutActivityLog(context));
```

Comment:
Calling the function here this way is intentional.  This way we can explicitly control what nodes get passed in.  It prevents accidentally selected nodes from being inadvertently passed in